packages/protocol/contracts/rrp/authorizers/Api3TokenLock.sol

```diff
@@ -0,0 +1,419 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.6;
+
+import "./Api3RequesterRrpAuthorizer.sol";
+import "./interfaces/IApi3Token.sol";
+import "./interfaces/IApi3TokenLock.sol";
+import "../../admin/MetaAdminnable.sol";
+import "../../admin/RankedAdminnable.sol";
+
+/// @title The contract used to lock API3 Tokens in order to gain access to Airnodes
+contract Api3TokenLock is RankedAdminnable, MetaAdminnable, IApi3TokenLock {
```

Comment:
Why do we need to inherit RankedAdminnable if we are already getting it's functionality thru MetaAdminnable? Am I missing something in the getRank() function? MetaAdminable overrides this function and then we also override it here so the Api3TokenLock.getRank() code will be the one executed I think.

Reply:
I think you're right. It's how it was done in the Api3RequesterRrpAuthorizer but that is inherriting two different RankedAdminnable contracts which is why we explicitly overrrided there. Not the case here